Content.Shared/Chemistry/Reagent/ReagentPrototype.cs

```diff
@@ -105,6 +105,12 @@ public sealed partial class ReagentPrototype : IPrototype, IInheritingPrototype
         [DataField]
         public bool Slippery;
 
+        /// <summary>
+        /// If this reagent is part fo a puddle does it evaporate
+        /// </summary>
+        [DataField]
+        public bool Evaporates;
```

Comment:
I think it would be better to make this a number for how fast it evaporates, instead of a bool. So different reagents can evaporate at different rates. 0 would be the same as false.

Reply:
It would simply be making it a 0,1 where 1 is true and 0 is false as adding that functionality would creep the scope beyond the magic variable decoupling. 

But yes this can be done